client/src/app/shared/models/poll/view-poll.ts

```diff
@@ -0,0 +1,141 @@
+import { ViewAssignment } from 'app/site/assignments/models/view-assignment';
+import { BaseProjectableViewModel } from 'app/site/base/base-projectable-view-model';
+import { BaseViewModel } from 'app/site/base/base-view-model';
+import { ProjectorElementBuildDeskriptor } from 'app/site/base/projectable';
+import { HasMeeting } from 'app/site/event-management/models/view-meeting';
+import { ViewMotion } from 'app/site/motions/models/view-motion';
+import { ViewGroup } from 'app/site/users/models/view-group';
+import { ViewUser } from 'app/site/users/models/view-user';
+import { Poll } from './poll';
+import {
+    MajorityMethodVerbose,
+    PollClassType,
+    PollClassTypeVerbose,
+    PollMethodVerbose,
+    PollPercentBaseVerbose,
+    PollStateChangeActionVerbose,
+    PollStateVerbose,
+    PollType,
+    PollTypeVerbose
+} from './poll-constants';
+import { ViewOption } from './view-option';
+
+export class ViewPoll extends BaseProjectableViewModel<Poll> {
+    public get poll(): Poll {
+        return this._model;
+    }
+    public static COLLECTION = Poll.COLLECTION;
+    protected _collection = Poll.COLLECTION;
+
+    public get pollClassType(): PollClassType {
```

Comment:
there are also polls without content objects. The return type should be optional.